earth_enterprise/src/common/gedbroot/SConscript

```diff
@@ -47,7 +47,7 @@ gedumpdbroot = env.executable('gedumpdbroot',
                                    'gecommon',
                                    'png12'])
 
-env.install('tools_bin', [gedumpdbroot])
+env.install('common_bin', [gedumpdbroot])
```

Comment:
Discussion: I think we should create a package for the tools, rather than moving tools over for this task.

Reply:
I don't see a need for a separate package - I think the opengee-common package is fine. However, this does seem a bit hacky to me. I think we should either update the installer to get the binaries out of the tools directory or remove all instances of `tools_bin`.

Reply:
I don't know what meaning we're ascribing to a package, but I prefer a package to have one piece of software, and dependencies for it.  If we don't have a clear meaning, and put tools that are not necessary for running a piece of software in a package that's its dependency, it makes it hard for a user who's not rather intimately familiar with the software and its packaging to find where a piece of software is, or to discover its existence. It also makes it harder for maintainers to know what's necessary for running a piece of software, and what isn't.

Reply:
This was resolved between the three of us in conversation.